hssqlppp/src/Database/HsSqlPpp/Pretty.lhs

```diff
@@ -1140,15 +1140,15 @@ syntax maybe should error instead of silently breaking
 
 > ppCsvOpts :: CsvOptions -> Doc
 > ppCsvOpts NewCsvOptions
->   { csvFilePath = path
+>   { csvFilePath = _path
```

Comment:
you can use `{..}` here instead and then use the full name of each field

Reply:
fixed it